Remove "private pages" in Excluding Pages (#2685)
withastro/docs: src/content/docs/en/core-concepts/routing.mdx

You can exclude pages, or even whole directories from being built by prefixing their names with an underscore (`_`).

> This allows you to temporarily disable pages, and also to co-locate tests, utilities, and components with their related pages, preventing them from being built into `.html` files and placed into the `dist/` directory.

Reviewer: ```suggestion
This allows you to disable page building, and also to co-locate tests, utilities, and components with their related pages, preventing them from being built into `.html` files and placed into the `dist/` directory.
```
I'm not sure temporary is the key thing here? (Although that is one particular use case.) But more accurately, it's that it will be ignored/not built. What about something like "disable page building"?

Author: We've said that in the previous sentence. I was thinking of a use case for preventing building, like "commenting out pages." I'm happy to remove this:
> This allows you to co-locate tests, utilities, and components with their related pages...

Reviewer: Maybe even 
```suggestion
This allows you to do things like temporarily disable pages, and also to co-locate tests, utilities, and components with their related pages, preventing them from being built into `.html` files and placed into the `dist/` directory.
```
I think it felt a little too much like "the only reason why"and not "a reason why"? Don't know why it didn't sit well with me. What do you think about just the modification above?

Author: I don't love the "do things," and we do list another reason why right after the first. It doesn't sound like a list because of the "and also". How does this sound:

```suggestion
You can use this to temporarily disable pages or to co-locate tests, utilities, and components with their related pages, preventing them from being built into `.html` files and placed into the `dist/` directory.
```
